langkit/support/langkit_support-token_data_handlers.adb

```diff
@@ -170,11 +168,37 @@ package body Langkit_Support.Token_Data_Handlers is
       Idx : Natural := 0;
       --  Index in T of the newline character that ends the currently processed
       --  line.
+
+      function Index
+        (Buffer             : Text_Type;
+         Char               : Wide_Wide_Character;
+         Start_Pos, End_Pos : Positive) return Natural;
```

Comment:
Given the errors in the travis, I guess you need to make `End_Pos` a Natural (not a positive) for empty buffers.